src/FAI3_backend/src/model.rs

```diff
@@ -170,24 +176,25 @@ pub fn get_model_metrics(model_id: u128) -> Metrics {
     check_cycles_before_action();
 
     MODELS.with(|models| {
-        let model = models
-            .borrow()
-            .get(&model_id)
-            .expect("Model not found");
-        return get_classifier_model_data(&model)
-            .metrics;
+        let model = models.borrow().get(&model_id).expect("Model not found");
+        return get_classifier_model_data(&model).metrics;
     })
 }
 
+/// Returns a model
+/// For limitations and data size, it won't return LLM data_points
+/// And it won't return LLM metrics history
 #[ic_cdk::query]
 pub fn get_model(model_id: u128) -> Model {
-    MODELS.with(|models| {
+    let model = MODELS.with(|models| {
         models
             .borrow()
             .get(&model_id)
             .expect("Model not found")
             .clone()
-    })
+    });
+
+    model.prune()
```

Comment:
I'm confused about this change, this should already be in main. Maybe we have a merge issue? Or Github is just displaying the diff wrong? I've seen other changes like this in the PR, things that already should exist in main.